build-configuration/pom.xml

```diff
@@ -197,6 +197,8 @@
       <version.spring5>5.2.4.RELEASE</version.spring5>
       <version.spring.session>2.3.0.RELEASE</version.spring.session>
       <version.micrometer>1.5.2</version.micrometer>
+      <version.cloudevents.sdk>1.3.0</version.cloudevents.sdk>
```

Comment:
Why are you not using sdk-java v2? @tristantarrant

Reply:
We try not to use non-final releases, and at the time 2.0.0 was only at milestone 3.
Now that RC1 is out, I'll upgrade.

Reply:
Cool!

Reply:
@slinkydeveloper I actually removed this property, because I'm not relying on the SDK to build the event. 

I build the event without the SDK because we are trying to remove the dependency on `jackson-databind`, which is a huge source of CVEs for us.

Reply:
I see, we hope to remove in the sdk too the jackson-databind dependency soon